addons/website/static/tests/tours/automatic_editor.js

```diff
@@ -80,10 +80,9 @@ wTourUtils.registerWebsitePreviewTour('automatic_editor_on_new_website', {
         trigger: 'button[name="button_choose_theme"]'
     },
     {
-        content: "Check that the editor is loaded",
-        trigger: 'iframe body.editor_enable',
+        content: "Check that the homepage is loaded",
+        trigger: ".o_website_preview[data-view-xmlid='website.homepage']",
```

Comment:
Maybe add a check to check if the `o_menu_systray` is there and is complete (with the user menu) ?
(If this hasn't already been done in another test)